BraintreeCore/src/main/java/com/braintreepayments/api/ShopperInsightsRequest.kt

```diff
@@ -0,0 +1,65 @@
+package com.braintreepayments.api
+
+import org.json.JSONObject
+
+/**
+ * Representation of a user phone number.
+ * @property countryCode The international country code for the shopper's phone number
+ * (e.g., "1" for the United States).
+ * @property nationalNumber The national segment of the shopper's phone number
+ * (excluding the country code).
+ */
+
+data class BuyerPhone(
```

Comment:
Can we move each data class to it's own file? I know they are small but we don't typically combine classes in files